cfgov/cfgov/urls.py

```diff
@@ -255,15 +255,15 @@
                 state=False),
 
     # If 'CCDB5_RELEASE' is false, include CCDB4 urls.
-    # Otherwise use CCDB5.
+    # Otherwise include CCDB5 urls
     flagged_url('CCDB5_RELEASE',
                 r'^data-research/consumer-complaints/',
                 include_if_app_enabled(
                     'complaintdatabase', 'complaintdatabase.urls'
                 ),
                 state=False,
-                fallback=TemplateView.as_view(
-                    template_name='ccdb5_landing_page.html'
+                fallback=include_if_app_enabled(
+                    'ccdb5_ui', 'ccdb5_ui.config.urls'
```

Comment:
This is fine as-is, but I wonder if it's worth reversing the conditionality so that it's a bit easier to follow (at least for me). Instead of flagging for falseness and falling back to truth, something like:

```python
 flagged_url('CCDB5_RELEASE',
             r'^data-research/consumer-complaints/',
             include_if_app_enabled(
                 'ccdb5_ui', 'ccdb5_ui.config.urls'
             ),
             fallback=include_if_app_enabled(
                 'complaintdatabase', 'complaintdatabase.urls'
             ),
)
```

It was reversed originally because included URL patterns weren't supported for fallbacks, but now they are :D